selfdrive/car/chrysler/values.py

```diff
@@ -19,16 +21,22 @@ class CAR:
   PACIFICA_2018 = "CHRYSLER PACIFICA 2018"  # includes 2017 Pacifica
   PACIFICA_2020 = "CHRYSLER PACIFICA 2020"
   JEEP_CHEROKEE = "JEEP GRAND CHEROKEE V6 2018"  # includes 2017 Trailhawk
-  JEEP_CHEROKEE_2019 = "JEEP GRAND CHEROKEE 2019" # includes 2020 Trailhawk
+  JEEP_CHEROKEE_2019 = "JEEP GRAND CHEROKEE 2019"  # includes 2020 Trailhawk
+
+
+@dataclass
+class ChryslerCarInfo(CarInfo):
+  package: str = "Adaptive Cruise"
+  harness: str = "fca"
```

Comment:
let's use enums here and the website can use the same definitions

Reply:
A global harness enum in car_definitions?

Reply:
In that case, we can remove some extra imports and code to fetch the footnotes and actually define an order if we put the footnotes in a global enum again

Reply:
yeah, think global enum makes the most sense